src/controllers/lostDoc/deleteItem.js

```diff
@@ -0,0 +1,11 @@
+import DeleteItem from '../../models/lost';
+
+export default async (req, res) =>{
+    try{
+        const deleteDoc = await DeleteItem.findByIdAndDelete(req.params._id)
+
+        return res.status(200).send({msg:`Document with number ${deleteDoc.documentNumber} deleted successfully`, doc});
+    }catch(error){
+        return res.status(400).send({error: 'No document with such Id found'});
```

Comment:
You have put every error in catch, If possible extract not found error and fix it in try not in catch

Reply:
lemme take care of it